docs/man/skuba-auth-login.1.md

```diff
@@ -0,0 +1,40 @@
+% skuba-auth-login(1) # skuba auth login - Login to a cluster
+
+# NAME
+login - Authenticate to a cluster and authorized with kubeconfig 
+
+# SYNOPSIS
+**login**
+[**--help**|**-h**] [**--server|-s**] [**--username|-u**]
+[**--password|-p**] [**--root-ca**|**-r**] [**--insecure**|**-k**]
+[**--cluster-name**|**-n**] [**--kubeconfig**|**-c**]
+*login* [--server https://<ip/fqdn>:<port>] [--username username] [--password password]
+
+# DESCRIPTION
+**login** lets you login to a cluster and authorized with kubeconfig
+
+# OPTIONS
+
+**--help, -h**
+  Print usage statement.
+
+**--server, -s**
+  (Required) The OIDC dex server url (https://<IP/FQDN>:<port>)
```

Comment:
Suggest to specify IP is controller plane.

`<IP/FQDN>` -> `<controller plane IP/FQDN>`

Reply:
updated!